### DISCUSSION TITLE
Monitoring Rootless container

### DISCUSSION THREAD

**Discussion by amikha33** [containers/podman#20309] [General]
### Feature request description

A clear and concise description of what the feature request is about.

**Comment by giuseppe** [comment]
it is not clear what you are asking for.  Please add more details

**Comment by amikha33** [comment]
as it's known podman is empowered in the rootless daemon mode what are the best tools for monitoring rootless containers?

**Reply by giuseppe** [reply]
you could delegate the monitoring to systemd.  Have you tried running a podman container using a systemd service?
